db/rdbms-schema/src/main/java/io/camunda/db/rdbms/LiquibaseScriptGenerator.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms;
+
+import com.github.vertical_blank.sqlformatter.SqlFormatter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Set;
+import liquibase.Liquibase;
+import liquibase.database.DatabaseFactory;
+import liquibase.resource.ClassLoaderResourceAccessor;
+import liquibase.sqlgenerator.SqlGeneratorFactory;
+
+public class LiquibaseScriptGenerator {
+
+  public static void main(final String[] args) throws Exception {
+    final var targetDir = args[0] + "/liquibase";
+    final var databases = Set.of("h2", "mysql", "postgresql");
+
+    for (final var database : databases) {
+      generateLiquibaseScript(
+          database, "db/changelog/rdbms-support/changelog-master.xml", targetDir, "master.sql");
+
+      generateLiquibaseScript(
+          database,
+          "db/changelog/rdbms-support/8_8_0/8.8.0_master.xml",
+          targetDir,
+          "8.8.0_master.sql");
+    }
+  }
+
+  public static void generateLiquibaseScript(
```

Comment:
Do we need tests for this?

Reply:
Yes. Added a test for the Script generation part.